@ecomplus/widget-martan/cms.config.js

```diff
@@ -0,0 +1,267 @@
+import { name } from './package.json'
+
+const label = 'Martan'
+const thumbnail = 'https://ik.imagekit.io/2wovc1fdm/storefront-widget.png'
+
+const martan = {
+  label: 'Configuração dos Widgets',
+  name: 'options',
+  widget: 'object',
+  hint: 'Para ajuda com a configuração use nossa documentação.(https://docs.martan.app/integracoes/ecomplus)',
+  fields: [
+    {
+      label: 'ID da loja',
+      hint: 'Obtenha aqui https://app.martan.app/settings',
+      name: 'store_id',
+      widget: 'number',
+      required: true
```

Comment:
```suggestion
      required: false
```
Enquanto a pessoa estiver editando outro widget, pode ficar apitando esse aqui, porque ele é requerido

Reply:
Tem certeza disso @matheusgnreis ?
O padrão é requerido mesmo, em quase todos os widgets os campos que são requeridos não ficam como opcionais mesmo, tipo aí https://github.com/ecomplus/storefront/blob/master/%40ecomplus/widget-tawkto/cms.config.js

Reply:
Eu acho que aqui os fields deveriam permanecer requeridos mesmo